core/transport/src/main/proto/grpc/file_system_master.proto

```diff
@@ -476,6 +476,15 @@ message UpdateUfsModePRequest {
   optional UpdateUfsModePOptions options = 2;
 }
 
+message GetStateLockHoldersPResponse {
+  repeated string threads = 1;
```

Comment:
We might as well return the thread id as well. Just in case we have duplicate thread names?